### PR TITLE
Add explicit `@return` type next to `#[ReturnTypeWillChange]`

### PR DESCRIPTION
**Pull request by nicolas-grekas**
Same as doctrine/collections#280

### CODE REVIEW

File: lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php [hunk 1]
```diff
@@ -82,6 +82,8 @@ protected function configure()
 
     /**
      * {@inheritdoc}
+     *
+     * @return int
```
Comment (greg0ire) [inline lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php:86]: Why is this needed for Symfony commands? This return type is not inherited from a native type like `Countable` or `ArrayIterator`, I thought this was only about native types. Isn't it?

Comment (nicolas-grekas) [reply lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php:86]: When used with `DebugClassLoader`, this saves a deprecation: `@return` means the same as `#[ReturnTypeWillChange]` to it.

Comment (greg0ire) [reply lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php:86]: Ok, I thought `DebugClassLoader` wouldn't need it in the first place, since no stubs are involved, only normal PHP files. Apparently that's not the case.

**Review by greg0ire — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @nicolas-grekas !
